pyirf/scripts/make_DL3.py

```diff
@@ -0,0 +1,338 @@
+"""Script to produce DL3 data from DL2 data and a configuration file.
+
+Is it initially thought as a clean start based on old code for reproducing
+EventDisplay DL3 data based on the latest release of the GADF format.
+
+Todo:
+- make some config arguments also CLI ones like in ctapipe-stage1-process
+
+
+"""
+
+# =========================================================================
+#                            MODULE IMPORTS
+# =========================================================================
+
+# PYTHON STANDARD LIBRARY
+
+import argparse
+import os
+
+# THIRD-PARTY MODULES
+
+import numpy as np
+import astropy.units as u
+from astropy.table import Table
+from astropy.coordinates.angle_utilities import angular_separation
+from gammapy.spectrum import cosmic_ray_flux, CrabSpectrum  # UPDATE TO LATEST
+
+# THIS PACKAGE
+
+from pyirf.io.io import load_config, read_FITS
+from pyirf.perf import (
+    CutsOptimisation,
+    CutsDiagnostic,
+    CutsApplicator,
+    IrfMaker,
+    SensitivityMaker,
+)
+
+
+def main():
+
+    # =========================================================================
+    #                   READ INPUT FROM CLI AND CONFIGURATION FILE
+    # =========================================================================
+
+    # INPUT FROM CLI
+
+    parser = argparse.ArgumentParser(description="Produce DL3 data from DL2.")
+
+    parser.add_argument(
+        "--config_file",
+        type=str,
+        required=True,
+        help="A configuration file like pyirf/resources/performance.yaml .",
+    )
+
+    parser.add_argument(
+        "--obs_time",
+        type=str,
+        required=True,
+        help="An observation time written as (value.unit), e.g. '50.h'",
```

Comment:
That's a really bad idea I think. Just require it to be in hours or separated by a space. Otherwise, what about e.g. half an  hour? `0.5.h`?

Reply:
Alternativelive, just let astropy do the parsing. `u.Quantity('50h')` works fine, `u.Quantity('0.5h')

Reply:
yep sure, I really don't know why I changed it...I set an empty space between value and unit

Reply:
DONE